From 6c69fc53dd7f8e34e2656160e2fbced85f432069 Mon Sep 17 00:00:00 2001
From: SallyQi <feng.qi@intel.com>
Date: Wed, 9 Dec 2020 23:43:42 +0800
Subject: [PATCH] [Media Common] Fix Gen12 Libva caps

Fix gen12 enc attributes height caps

(cherry picked from commit ffbb966ade01fa0566ba36513f3ce099e8562f0e)
---
 media_driver/linux/gen12/ddi/media_libva_caps_g12.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/media_driver/linux/gen12/ddi/media_libva_caps_g12.cpp b/media_driver/linux/gen12/ddi/media_libva_caps_g12.cpp
index bf80a169..5d3cfa64 100644
--- a/media_driver/linux/gen12/ddi/media_libva_caps_g12.cpp
+++ b/media_driver/linux/gen12/ddi/media_libva_caps_g12.cpp
@@ -1599,7 +1599,7 @@ VAStatus MediaLibvaCapsG12::CreateEncAttributes(
 
     attrib.type = VAConfigAttribMaxPictureHeight;
     GetPlatformSpecificAttrib(profile, entrypoint,
-        VAConfigAttribMaxPictureWidth, &attrib.value);
+        VAConfigAttribMaxPictureHeight, &attrib.value);
     (*attribList)[attrib.type] = attrib.value;
 
     attrib.type = VAConfigAttribEncJPEG;
-- 
2.29.2

